library/aes.c

```diff
@@ -1172,6 +1172,7 @@ int mbedtls_internal_aes_encrypt( mbedtls_aes_context *ctx,
     } while( stop_mark == 0 );
 
     // Calculate final AES round + dummy rounds
+    mbedtls_platform_random_delay();
```

Comment:
I thought about it again(during masking implementaion) , i think we shouldn't add it . Please remove

Reply:
Only this one?

Reply:
Only this one